recipes/build/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "build" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/build-{{ version }}.tar.gz
```

Comment:
Are we sure this package is from the upstream source? I was checking on pypi and the project home linked to a fork of pypa/build.

Reply:
It is a mess but yeah. It is like that.

Reply:
BTW, this is the PyPA blessed package for pep517 builds.